### ISSUE TITLE
incorporate Grunt & SASS for testing and minification/awesomeness

### ISSUE THREAD

**Issue by mapsam** [codeforamerica/RVA-FAMIS#8]
Would love to up the caliber of our development process by incorporating some pre-processors into the mix. I have a quick and easy-to-install grunt starter that we can work with that comes with some js linting and SASS attachments.

Thoughts @bengolder @esmithayer?

**Comment by bengolder** [comment]
I've used [gulp](http://gulpjs.com/) and [less](http://lesscss.org/) a bunch. I love using them and could set them up quickly. To work with Grunt & SASS, I would need a brief intro before contributing. But I'm happy either way.

**Comment by mapsam** [comment]
I've used LESS and have only been using SASS recently to change it up. Have no resounding opinion with one over the other.

Haven't used gulp but am totally open to learning it. Seems like gulp can be really good for larger projects since you can chain together processes pretty well, right? I don't think this step is necessary for Friday, but we should use it as a way to explore quick dev and deploy options!

**Comment by esmithayer** [comment]
I've also used less, but I'd definitely be interested in learning sass.  No strong opinion on gulp vs grunt, but if you've got something ready to go with grunt and sass, that sounds good to me.

**Comment by mapsam** [comment]
Sounds like we should try both out over the next couple weeks to see what we like using best? Gulp is definitely the system with potential for larger projects, whereas Grunt seems best for small things (according to the single article I read about them both).

**Comment by bengolder** [comment]
Yeah let's try both next week!

**Comment by bengolder** [comment]
@svmatthews can you link the article?

**Comment by mapsam** [comment]
Yup! quick presentation by Mark Dalgleish describes the overall distinction pretty well in my mind - http://markdalgleish.github.io/presentation-build-wars-gulp-vs-grunt/

**Comment by bengolder** [comment]
So I've used both Grunt and Gulp, and my experience is that Gulp is easier to learn, extend, and understand. Gulp is not particularly aimed at large projects (but Grunt becomes unwieldy in such situations) and feels easy in a small project. I'd recommend going with Gulp, and I'd be happy to give you both an intro. Here's an example file: https://gist.github.com/bengolder/7d3c21ada5d84f036f56

**Comment by mapsam** [comment]
:+1: :dancers:
